.travis.yml

```diff
@@ -6,7 +6,8 @@ node_js:
 env:
     - CXX=g++-4.8
 
-sudo: false
+dist: trusty
+sudo: required
```

Comment:
It's required to use ubuntu trusty https://docs.travis-ci.com/user/ci-environment/